src/OpenLoco/Drawing/SoftwareDrawingEngine.cpp

```diff
@@ -180,16 +176,19 @@ namespace OpenLoco::Drawing
 
     void SoftwareDrawingEngine::updatePalette(const PaletteEntry* entries, int32_t index, int32_t count)
     {
-        SDL_Color base[256];
-        for (int i = 0; i < 256; i++)
+        assert(index + count < 256);
```

Comment:
I was looking at how this was used and there is only actually 236 colours ever in use not the full 256 so thought we may as well enforce that.